tests/xfstests/enable_kdump.pm

```diff
@@ -28,7 +28,7 @@ sub run {
     # Also panic when softlockup
     # workaround bsc#1104778, skip s390x in 12SP4
     assert_script_run('echo "kernel.softlockup_panic = 1" >> /etc/sysctl.conf');
-    my $output = script_output('sysctl -p');
+    my $output = script_output('sysctl -p', 10, 1, 0);
```

Comment:
With this you've just set $wait to 10 seconds, for other parameters you need to pass them as key-value pair:
```perl
script_output('sysctl -p', 10, type_command => 1, proceed_on_failure => 1)
```
Or do you try to achieve something else here?

Reply:
Here I wanna set proceed_on_failure => 0 to continue test process when "sysctl -p" has any problem. Please see what I describe in PR. To solve this fail https://openqa.suse.de/tests/2023999#step/enable_kdump/10

I just want a soft fail, not a fail.

Reply:
Then, please take a look on [api documentation](http://open.qa/api/testapi/#_script_output), only 2 first parameters will work here, other 2 will be basically ignored: https://github.com/os-autoinst/os-autoinst/blob/master/distribution.pm#L195 as we check for 2 keys in the hash. I also believe that we need to set proceed_on_failure => 1 and not to 0 as you stated above to reach this behavior. Hope it helps.

Reply:
I got what you mean, does proceed_on_failure => 1 means test will keep going even script inside script_output failed? But way my case failed by this, I don't understand this point.

Reply:
Yes, if you will call it like `script_output('sysctl -p', 10, proceed_on_failure => 1)` test won't fail even if script returns non-zero code. In your code you are passing parameters as a integer, but it should be keymap.

Reply:
Got it, code changed. Thanks! @rwx788  I was thinking the default value is proceed_on_failure => 1